src/harshadBJun20/Assignment20/PrintReverseIntegerNumber.java

```diff
@@ -0,0 +1,22 @@
+package harshadBJun20.Assignment20;
+/*Program 3: WAP to revers number (Consider Input as Integer not as String).
+Example:-
+Input:- 12345
+Output:- 54321*/
+public class PrintReverseIntegerNumber {
+	void printReverseNumbers(int num) {
+		int seperateNum=0;
+		System.out.println("Reverse order Number: ");
+		while(num>0) {
+			seperateNum = num % 10;
+			num = num / 10;
+			System.out.print(seperateNum);
```

Comment:
You are just printing the reverse order of the given number, Please implement integer reverse logic - which is just a small extension to your logic.

Reply:
Hi Aastha,
Changes as advised is done. Please review.